src/style.css

```diff
@@ -1,3 +1,71 @@
 body {
   margin: 0;
 }
+
+:root {
+  --activ-color: #00acdc;
+}
+
+.header {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+}
+
+.header__menu {
+  margin: 0 50px;
+}
+
+.menu {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.menu__logo {
+  flex-grow: 1;
+  height: 40px;
+  width: 40px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__list-item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  height: 60px;
+  text-decoration: none;
+  color: black;
+}
+
+.nav__link_blue {
+  color: var(--activ-color);
+}
+
+.nav__link:hover {
+  color: var(--activ-color);
+}
+
+.is-active {
+  position: relative;
+}
+
+.is-active::after {
+  content: '';
+  height: 4px;
+  width: 100%;
+  background-color: var(--activ-color);
+  border-radius: 8px;
+  position: absolute;
```

Comment:
Pay attention to the CSS properties order 
https://9elements.com/css-rule-order/